docs/taskruns.md

```diff
@@ -7,33 +7,39 @@ weight: 300
 
 # `TaskRuns`
 
-- [Overview](#taskruns)
+<!-- toc -->
+- [Overview](#overview)
 - [Configuring a `TaskRun`](#configuring-a-taskrun)
   - [Specifying the target `Task`](#specifying-the-target-task)
   - [Tekton Bundles](#tekton-bundles)
   - [Specifying `Parameters`](#specifying-parameters)
+    - [Implicit Parameters](#implicit-parameters)
+    - [Extra Parameters](#extra-parameters)
   - [Specifying `Resources`](#specifying-resources)
-  - [Specifying `ServiceAccount` credentials](#specifying-serviceaccount-credentials)
+  - [Specifying `Resource` limits](#specifying-resource-limits)
   - [Specifying a `Pod` template](#specifying-a-pod-template)
   - [Specifying `Workspaces`](#specifying-workspaces)
   - [Specifying `Sidecars`](#specifying-sidecars)
   - [Specifying `LimitRange` values](#specifying-limitrange-values)
-  - [Configuring the failure timeout](#configuring-the-failure-timeout)
+- [Configuring the failure timeout](#configuring-the-failure-timeout)
+  - [Specifying `ServiceAccount` credentials](#specifying-serviceaccount-credentials)
 - [Monitoring execution status](#monitoring-execution-status)
   - [Monitoring `Steps`](#monitoring-steps)
+  - [Steps](#steps)
   - [Monitoring `Results`](#monitoring-results)
 - [Cancelling a `TaskRun`](#cancelling-a-taskrun)
-- [Debugging a `TaskRun`](#debugging-a-taskrun)
-- [Events](events.md#taskruns)
-- [Running a TaskRun Hermetically](hermetic.md)
+  - [Debugging a `TaskRun`](#debugging-a-taskrun)
```

Comment:
Also, please move this out of the `Cancelling a TaskRun` section, such that, both are on same level:

Cancelling a TaskRun
Debugging a TaskRun